be/src/Project/Services/DownloadCenter/DownloadCenter.Api/v1/Controllers/CategoriesController.cs

```diff
@@ -159,11 +159,11 @@ public async Task<IActionResult> Get(Guid? id)
             };
 
             var validator = new GetCategoryModelValidator();
-            var validationResult = await validator.ValidateAsync(serviceModel);
```

Comment:
**suggestion:** we prefer async methods. If they are available or used, let's leave them